ajax/libs/mui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "mui",
+  "filename": "js/mui.min.js",
+  "description": "A effective front-end framework closest to the native app",
```

Comment:
I don't have a good candidate yet but I think this one may be too long and not accurate enough, we may need more time on this, or maybe just write a individual description for it, not translate it.